Makefile

```diff
@@ -144,7 +144,7 @@ ifeq (, $(shell which kind))
 	KIND_GEN_TMP_DIR=$$(mktemp -d) ;\
 	cd $$KIND_GEN_TMP_DIR ;\
 	go mod init tmp ;\
-	GO111MODULE="on" go get sigs.k8s.io/kind@v0.9.0 ;\
+	GO111MODULE="on" go get sigs.k8s.io/kind@v0.11.1 ;\
```

Comment:
What do you think about changing the whole target to something like this?

```Makefile
KIND_VERSION=v0.11.1

kind:
ifneq ($(KIND_VERSION), $(shell kind version | cut -d' ' -f2))
	go install sigs.k8s.io/kind@$(KIND_VERSION)
KIND=$(GOBIN)/kind
else
KIND=$(shell which kind)
endif
```

Reply:
If you have kind installed other than with Go (e.g. with Homebrew in macOS) and the version mismatches the expected one, then this will not help, but neither it would that other implementation that just checks for `which kind`.

Reply:
It looks good to me, I will change it and test it locally.